### PR TITLE
Add `social-media/followers` profile with Twitter map

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR adds new `social-media/followers` profile with mock and twitter provider maps.

## Motivation and Context

`social-media/followers` profile with `GetFollowers` use-case allows users to read followers of social media profile.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: grid/social-media/followers/profile.supr [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      username string
+      """
+      Image url
+      Follower profile image url.
+      """
+      imageUrl string
+    } ]
+  }
+
+  error {
```
Comment (jnv) [inline grid/social-media/followers/profile.supr:56]: I suggest there should be an error model suitable for rate limiting which a) is easy to capture b) includes details about the limit.

File: grid/social-media/followers/profile.supr [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Get Followers
+Retrieve paginated list of social-media profile followers.
+"""
+usecase GetFollowers safe {
+  input {
+    """
+    Profile ID
+    Identification of profile to retrieve posts from; may be optional with some providers.
+    """
+    profileId string
```
Comment (jnv) [inline grid/social-media/followers/profile.supr:14]: Similar to lookups, we should allow fetching that with username, but that can be added later.

**Review by jnv — COMMENTED** [review]
Looks good, I just miss some data.

**Review by martinalbert — APPROVED** [review]

File: grid/social-media/followers/profile.supr (hunk 1 above)
Comment (martinalbert) [reply grid/social-media/followers/profile.supr:56]: What about new property `retryAfter`?

File: grid/social-media/followers/profile.supr [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    """
+    followers [ {
+      """
+      Follower ID
+      Unique follower identification.
+      """
+      id string
+      """
+      Username
+      Username of the follower.
```
[the remark this answers is not in the document]
Comment (janhalama) [reply grid/social-media/followers/profile.supr:45]: I will add it in followup PR and merge this now.

File: grid/social-media/followers/profile.supr (hunk 1 above)
Comment (janhalama) [reply grid/social-media/followers/profile.supr:56]: we can get that info `retryAfter` from headers 👍, I will add it in separate PR

**Pull request merged**